driver.go

```diff
@@ -43,4 +43,9 @@ type Driver interface {
 	//
 	// Since: 2.5
 	SetDisableScreenBlanking(bool)
+
+	// CallFromGoroutine provides a way to queue a function that is running on a goroutine back to the main thread.
+	// This is required when background tasks want to execute code safely in the graphical context.
+	// Since: 2.6
+	CallFromGoroutine(func())
```

Comment:
Slightly nitpicky, but I wonder if the name `RunFromGoroutine` that was discussed is better. To me, `CallFromGoroutine` could be interpreted as "it will call the supplied func from a new goroutine" ie a synonym for `go f()`. I'm sure it's fine either way though, just something to think about.

Reply:
Hmm, now that I think about it, the `RunFromGoroutine` name could have the same misinterpretation perhaps. Not sure it matters

Reply:
Yeah, I think both have that misinterpretation. I'm not a huge fan of it for that exact reason. I think we need something different.

Reply:
I think RunOnMain is a better name.

Reply:
We cannot use the `Run` prefix, particularly not in a situation that implies an app can be run from a goroutine.

I could use `CallOnMain` - but then there is the slight complication that "main thread" is not part of our lexicon in the public API nor the Go familiar terms either.

Reply:
This is a new API on the public driver interface though, even if we introduce another one later, likely at the root package. I suppose we can amend the name of this one at that point, since we don't promise no breaking changes on develop

Reply:
Agreed, that is what I meant by "with a change to driver if required."